pyaedt/application/Design.py

```diff
@@ -2242,13 +2242,15 @@ def import_dataset1d(self, filename, dsname=None, is_project_dataset=True):
         )
 
     @pyaedt_function_handler()
-    def import_dataset3d(self, filename, dsname=None):
+    def import_dataset3d(self, filename, encoding="utf-8-sig", dsname=None):
```

Comment:
To keep back compatibility better to add the new args at the end of the method

```suggestion
    def import_dataset3d(self, filename, dsname=None, encoding="utf-8-sig"):
```